purchase_open_qty/README.rst

```diff
@@ -16,18 +16,7 @@ To use this module, you need to go to Purchase Orders and select the new filters
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/142/10.0
-
-Bug Tracker
-===========
-
-Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
-check there if your issue has already been reported. If you spotted it first,
-help us smash it by providing detailed and welcomed feedback.
-
-Credits
-=======
+   :target: https://runbot.odoo-community.org/runbot/142/11.0
```

Comment:
Check the [OCA template](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst) and adapt this README file to the smaller pieces at the (new) `readme` directory